cli_tools/diagnostics/rdp_status.ps1

```diff
@@ -25,6 +33,14 @@ $items= @(
   @{
     description="Verify that MTU size is no greater than 1460, expect [MTU <= 1460, Interface:Ethernet]:";
     cmd="netsh interface ipv4 show subinterfaces"
+  },
+  @{
+    description="Verify that client-server seecurity negotiation is set to default value, expect [SecurityLayer REG_DWORD 0x1]:";
```

Comment:
This is at least incorrect for Windows 2022 as it's set to 2.

I would also caution again stating "set to default value", If there is an expected value it's better to state what it is and why.

For these items I'd suggest linking to this:
https://learn.microsoft.com/en-us/windows-hardware/customize/desktop/unattend/microsoft-windows-terminalservices-rdp-winstationextensions-securitylayer

All 3 values this setting can have are correct depending on the situation. So for this type of setting we should absolutely collect it to understand it's configuration, but there should be no expected value beyond it being between 0 and 3. If you want you can give a short statement to summarize what each value is so going to the link isn't required.

0=Legacy RDP protocol, 1=Negotiate Legacy/TLS, 2=TLS required.

Reply:
Please I have added this as well, I did not realise the changes early on. My bad